pkg/apis/addtoscheme_logging_v1.go

```diff
@@ -0,0 +1,10 @@
+package apis
+
+import (
+	v1 "github.com/openshift/elasticsearch-operator/pkg/apis/logging/v1"
+)
+
+func init() {
+	// Register the types with the Scheme so the components can map objects to GroupVersionKinds and back
+	AddToSchemes = append(AddToSchemes, v1.SchemeBuilder.AddToScheme)
```

Comment:
Would this allow us to watch multiple kinds?

Reply:
we would do that in the controller, but yes.

Reply:
https://github.com/openshift/elasticsearch-operator/pull/122/files#diff-9e63f7b0791edcb4a53c944552b88970R105